StyleText/engine/text_drawers.py

```diff
@@ -23,7 +23,7 @@ def load_fonts(self, fonts_config):
 
     def get_valid_height(self, font_path):
         font = ImageFont.truetype(font_path, self.height - 4)
-        _, font_height = font.getsize(self.char_list)
+        _ , _ , _ , font_height = font.getbbox(self.char_list)
```

Comment:
previous: 
width, height = font.getsize("Hello world")

Now:
left, top, right, bottom = font.getbbox("Hello world")
width, height = right - left, bottom - top

refer:
https://pillow.readthedocs.io/en/stable/releasenotes/9.2.0.html#font-size-and-offset-methods